src/redux/books/books.js

```diff
@@ -1,25 +1,50 @@
+import { newBook, getBooks, deleteBook } from '../../api/api';
+
 const ADD_BOOK = 'bookStore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const GET_BOOKS = 'bookStore/books/GET_BOOKS';
 
 const initialState = [];
 
-export const addBook = (payload) => ({
-  type: ADD_BOOK,
-  payload,
-});
+export const addBook = (book) => async (dispatch) => {
+  await newBook(book);
+  dispatch({
+    type: ADD_BOOK,
+    payload: book,
+  });
+};
+
+export const removeBook = (book) => async (dispatch) => {
```

Comment:
The remove book function does not work as expected. The user needs to refresh the page to see the effect of removal functionality. Please make sure that the item is removed and the list is updated in real-time.